CMakeLists.txt

```diff
@@ -306,11 +313,15 @@ set(CMAKE_CXX_FLAGS_DEBUG   " -DDEBUG -g3 -ggdb3")
 
 if (CMAKE_SYSTEM_NAME STREQUAL "Linux")
   add_definitions(-DOS_LINUX)
+  SET(BUILD_BLE_SUPPORT OFF)
 elseif(CMAKE_SYSTEM_NAME STREQUAL "QNX")
   add_definitions(-DOS_QNX)
   SET(BUILD_BT_SUPPORT OFF)
+  SET(BUILD_BLE_SUPPORT OFF)
```

Comment:
Remove that

Reply:
fixed in 8c504abbb37320d77c18a8662a47b5a61942b7cf